cdc/redo/writer/writer.go

```diff
@@ -433,7 +473,18 @@ func (l *LogWriter) DeleteAllLogs(ctx context.Context) error {
 	}
 	// after delete logs, rm the LogWriter since it is already closed
 	l.cleanUpLogWriter()
-	return nil
+
+	// write a marker to s3, since other process get the remove changefeed job async,
+	// may still write some logs after owner delete the log
+	return l.writeDeletedMarkerToS3(ctx)
+}
+
+func (l *LogWriter) getDeletedChangefeedMarker() string {
+	return fmt.Sprintf("delete_%s", l.cfg.ChangeFeedID)
+}
+
+func (l *LogWriter) writeDeletedMarkerToS3(ctx context.Context) error {
+	return cerror.WrapError(cerror.ErrS3StorageAPI, l.storage.WriteFile(ctx, l.getDeletedChangefeedMarker(), []byte("D")))
 }
```

Comment:
Is the delete marker file written to S3, and the redo log dir in S3 is not deleted after changefeed is removed?

Reply:
yes, the bucket is still there, no extra cost. but currently we do not delete the bucket even without the marker, just delete the object